backend/storage/discussion-board.sql

```diff
@@ -0,0 +1,82 @@
+CREATE TYPE position_types AS ENUM ('Student', 'TA', 'Instructor');
+
+CREATE TABLE users (
```

Comment:
one nice thing about piazza is that it keeps track of contributions!
we should probably have a field for this